{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.html

```diff
@@ -1,30 +1,20 @@
 {% raw %}<!DOCTYPE html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!-->
 <html class="no-js">
-  <!--<![endif]-->
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>{%block title%} {%endblock title%}</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
+    
     {% load js pipeline %}
```

Comment:
Sorry, I wasn't clear. You can load the tags of multiple libraries in one statement. One is `js` - that is from `djangojs` and one is `pipeline` which is from `django-pipeline`.

So we need to change it from
`{% load js pipeline %}`
to
`{% load pipeline %}`